js/hitbtc3.js

```diff
@@ -38,8 +38,9 @@ module.exports = class hitbtc3 extends Exchange {
                 'fetchFundingRateHistory': true,
                 'fetchFundingRates': false,
                 'fetchIndexOHLCV': undefined,
-                'fetchLeverageTiers': undefined,
-                'fetchMarketLeverageTiers': undefined,
+                'fetchLeverage': true,
+                'fetchLeverageTiers': false,
```

Comment:
@Dan-krm These can be hard to find sometimes, and can be hidden in other data, so be careful when marking this method as false